testScene.py

```
@@ -0,0 +1,10 @@
+from manim import *
```

Ruff (0.12.2)

`from manim import *` used; unable to detect undefined names

(F403)